README.md

```diff
@@ -11,157 +18,79 @@ Client:
 
-### Webpack Production mode
-
-In the production mode, we will have only 1 server running. All the client side code will be bundled into static files using webpack and it will be served by the Node.js/Express application.
-
-## Quick Start
-
-```bash
+```
 # Clone the repository
-git clone <this.repo/>
+git clone https://github.com/sandstreamdev/eoc.git
 
 # Go inside the directory
 cd [project-folder]
 
 # Install dependencies
 npm install
 
-# Start development server
+# Run database
+monogod
+
+# Start the development server
 npm run dev
 
 # Build for production
 npm run build
-
-# Start production server
-npm start
 ```
 
-## Avalible scripts:
-
-- "build" -> Builds static assets in production mode.
-- "clear-demo": cleans the database from demo users' changes
-- "docker:build:client": "docker build src/client",
-- "docker:build:server": "docker build src/server",
-- "start" -> Runs "build" script and serves static files from server directory
-- "client" -> Runs Client layer of application at given port.
-- "server" -> Runs Server layer of application at given port.
-- "dev": -> Runs Client and Server layers at once.
-- "lint": -> Lints all the js, json files inside src directory.
-- "prettier": -> Prettifies all the .js, .scss files inside src directory.
-- "migrate:up" -> Run up db migration
-- "migrate:down" -> Run down db migration
-
-## Documentation
-
-### Docker
-
-To run project within Docker containers simply clone this repo, install Docker on your machine and the in the root foolder of this app run
-`docker-compose up`. There will be three containers running, one for the client, second for the server, third for the database. The app will be available under `localhost:3000` .
-
-### Stylelint
-
-To use stylelint you need run. `npm run stylelint`
-
-### Run MongoDB locally
-
-To run MongoDB locally you need to clone all the files from this repository and then in the root directory run in terminal: <br/> `sudo mongod`
-By default MongoDB server will run at `localhost:27017`.
-There might be others mongodb proccesses runnig at this port. To stop all runing mongodb services type in your terminal: <br/> `sudo service mongodb stop`. Than simply rerun `sudo mongod`.
-
+## Seeding database for development
 
-### DB migrations
+At first run of the app, log in and check your user id in the database under eoc/users directory. Put that user ID, generated by mongo to .env file under USER_ID variable and save the file. Then simply run npm run seed. Seeding is not working? Probably USER_ID in your database is not matching with USER_ID in your .env file. Be aware that if you clean up the whole database with users collection, your USER_ID delivered by mongo will be different compared to the previous session. TIP When playing with data, always clean all the collections omitting users collection, so you don't have to deal with USER_ID configuration over and over again.
```

Comment:
should be split into subsections. If you read it it looks like there should be new paragraph here:
- `Seeding is not working...`
- `TIP`